app/views/mailing_list/steps/_degree_status.html.erb

```diff
@@ -10,6 +10,11 @@
   f.object.degree_status_options,
   :id,
   ->(option) { degree_status_text(option) },
```

Comment:
Making a quick note, since I've introduced the `format_option_value` helper, I think we can get rid of this `degree_status_text` now as the new helper is much more generic and cleaner